

geometry_msgs/PoseStamped pose # The grasp pose

geometry_msgs/Vector3 approach # The grasp approach direction

std_msgs/Float32 score # Score assigned to the grasp by the classifier

gpd/GraspConfig grasp # gpd default grasp msg
